packages/waku/src/config.ts

```diff
@@ -48,6 +48,8 @@ export interface Config {
    * ]
    */
   middleware?: () => Promise<{ default: Middleware }>[];
+
+  typegen?: boolean;
```

Comment:
I don't think we need to make it configurable for now.

Reply:
In the future we may want to configure vite plugins.

Reply:
sounds good and yes this allows removing the temp vite server in the vite-loader